frontend/src/components/QuickAdd.tsx

```diff
@@ -79,27 +73,45 @@ export const QuickAdd = ({ addIssueActivity }) => {
     setActivity(activity);
   };
 
+  const getSearchClasses = () => {
+    let classes = "col-2 quick-add-input ";
+    if (search != "") classes += issue ? "valid" : "invalid";
+    return classes;
+  };
+
+  const getValidationIconSrc = () => {
+    let src = "";
+    if (search != "") src = issue ? check : x;
+    return src;
+  };
+
   return (
     <div className="row">
       <h2>Quick add:</h2>
-
       <input
         aria-label="Issue"
         id="input-issue"
-        className={classes}
+        className={getSearchClasses()}
         type="number"
         min={0}
-        onChange={(e) => setSearch(e.target.value)}
+        onChange={(e) => {
+          setSearch(e.target.value);
+          setIssue(undefined);
+        }}
         placeholder="Type issue number..."
         title={(issue && issue.subject) || ""}
       />
+      <img
+        className={search == "" ? "validation-icon hiden" : "validation-icon"}
```

Comment:
always use the triple `===` for validating, not the double `==`

Reply:
in this case the type is the same, string, so there is no need for a triple comparison https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality